packages/plugin-ext/src/main/browser/plugin-icon-theme-service.ts

```diff
@@ -329,8 +330,10 @@ export class PluginIconTheme extends PluginIconThemeDefinition implements IconTh
             return undefined;
         }
         const iconUri = this.locationUri.resolve(iconPath);
-        const relativePath = this.packageUri.path.relative(iconUri.path.normalize());
-        return relativePath && `url('hostedPlugin/${this.pluginId}/${encodeURIComponent(relativePath.normalize().toString())}')`;
+        const relativePath = this.packageRootUri.path.relative(iconUri.path.normalize());
+        return relativePath && `url('${new Endpoint({
+            path: `hostedPlugin/${this.pluginId}/${encodeURIComponent(relativePath.normalize().toString())}`
+        }).getRestUrl().toString()}')`;
```

Comment:
The last `toString()` should be `toStirng(true)`, shouldn't it? Otherwise, whitespace will be `%20` in your relative path.

Reply:
or we should get rid of `encodeURIComponent`, since we go through vscode uri encoding?

Reply:
No, I was wrong. Ignore this 👆